src/base/static/components/app.js

```diff
@@ -56,7 +56,7 @@ browserUpdate({
 
 const TemplateContainer = styled("div")(props => ({
   position: "relative",
-  overflow: props.layout === "desktop" ? "hidden" : "visible",
+  overflow: props.layout === "desktop" ? "auto" : "visible",
```

Comment:
@goldpbear I'm not sure why the scrollbars were hidden on desktop, but making this `auto` fixes a scrollbar issue on the dashboards page.

It doesn't seem like this change affects the map or list views, but lmk if I'm missing something.

Reply:
I think that's ok-- I can't remember why the overflow was hidden, but in testing setting it to `auto` seems to be fine.